lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -172,6 +174,11 @@ const FeatureInfoSection = React.createClass({
                             <When condition={reactInfo.showRawData || !this.hasTemplate()}>
                                 <If condition={reactInfo.hasRawData}>
                                     {reactInfo.rawData}
+                                    <If condition={reactInfo.timeSeriesChart}>
+                                        <div className="time-series-chart">
```

Comment:
Could you change it to `Styles.timeSeriesChart` and define `@include empty-module("time-series-chart") in `feature-info-section.scss` then :). That means it'll be namespaced the same as all the other classes.